### DISCUSSION TITLE
Typing an accent key breaks the input

### DISCUSSION THREAD

**Discussion by sballano** [ghostty-org/ghostty#4223] [Q&A] [answered]
Pressing most of the accent keys like ', ¨, ^, ` that are used for languages like spanish or french seem to break the input, at least on my setup...

I'm using Ubuntu 24.04 in gnome with a Spanish keyboard

[screencap.webm](https://github.com/user-attachments/assets/6565bd63-92fd-42ed-97b2-cccd50e9d75e)

Curious if somebody else is experiencing this, I haven't seen any open issue/discussion about this

**Comment by JulienMasgonty** [comment]
Experiencing the same thing here

I'm on Ubuntu 24

My config below just in case i messed with something i shouldn't have (but well, it's pretty basic)

```
gtk-tabs-location = hidden
scrollback-limit = 10_000
window-theme = ghostty
clipboard-read = allow
clipboard-paste-protection = false
window-decoration = false
shell-integration = zsh  
font-family = "Geist Mono"
font-size = 14
font-feature = -liga
window-padding-color = background
theme = vesper
background-opacity = 0.6
```

Only way around it right now is to force close the terminal and reopen it (pretty annoying when i'm in the middle of writing something in nvim and missclick the "^" instead of p, which is right next to it on my azerty keyboard)

Other terminals work fine

**Reply by JulienMasgonty** [reply]
> Experiencing the same thing here
> 
> I'm on Ubuntu 24
> 
> My config below just in case i messed with something i shouldn't have (but well, it's pretty basic)
> 
> ```
> gtk-tabs-location = hidden
> scrollback-limit = 10_000
> window-theme = ghostty
> clipboard-read = allow
> clipboard-paste-protection = false
> window-decoration = false
> shell-integration = zsh  
> font-family = "Geist Mono"
> font-size = 14
> font-feature = -liga
> window-padding-color = background
> theme = vesper
> background-opacity = 0.6
> ```
> 
> Only way around it right now is to force close the terminal and reopen it (pretty annoying when i'm in the middle of writing something in nvim and missclick the "^" instead of p, which is right next to it on my azerty keyboard)
> 
> Other terminals work fine

Just saw this issue reagrding this  : #3567

**Comment by mitchellh** [comment, marked as answer]
This is fixed by #5324
